### PR TITLE
Fix #6539

### PR DESCRIPTION
**Pull request by tharikaGitHub**
Fixes https://github.com/wso2/product-apim/issues/6539

### CODE REVIEW

File: features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/site/public/locales/en.json [hunk 1]
```diff
@@ -119,7 +119,7 @@
   "Apis.Details.Configuration.components.APISecurity.TranportLevel.certificate.alias.error": "Something went wrong while adding the API certificate",
   "Apis.Details.Configuration.components.APISecurity.TranportLevel.certificate.delete.error": "Error while deleting certificate",
   "Apis.Details.Configuration.components.APISecurity.TranportLevel.certificate.delete.success": "Certificate Deleted Successfully",
-  "Apis.Details.Configuration.components.APISecurity.allOptional": "Please select at least one API security level mandatory!",
+  "Apis.Details.Configuration.components.APISecurity.allOptional": "Please select at least one API security level as mandatory!",
```
Comment (bhathiya) [inline features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/site/public/locales/en.json]:
```suggestion
  "Apis.Details.Configuration.components.APISecurity.allOptional": "Please select at least one API security level as 'Mandatory'!",
```

File: features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/Configuration/components/APISecurity/APISecurity.jsx [hunk 2]
```diff
@@ -92,7 +92,7 @@ export default function APISecurity(props) {
                 <Typography className={classes.bottomSpace}>
                     <FormattedMessage
                         id='Apis.Details.Configuration.components.APISecurity.allOptional'
-                        defaultMessage='Please select at least one API security level mandatory!'
+                        defaultMessage='Please select at least one API security level as mandatory!'
```
Comment (bhathiya) [inline features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/Configuration/components/APISecurity/APISecurity.jsx]:
```suggestion
                        defaultMessage='Please select at least one API security level as 'Mandatory'!'
```

**Comment by bhathiya** [comment]
@tharikaGitHub WDYT about the above suggestions? Do they make sense?

**Comment by tharikaGitHub** [comment]
@bhathiya yes those suggestions make sense. I'll commit the changes.

**Pull request merged**